tests/testthat/_snaps/dm.md

```diff
@@ -128,6 +128,7 @@
       Tables: `tf_1`, `tf_2`, `tf_3`, `tf_4`, `tf_5`, `tf_6`
       Columns: 20
       Primary keys: 6
+      Unique keys: 1
```

Comment:
Why are we now seeing explicit unique keys here?

Reply:
Was quick to add and is easy to remove. Now that the implicit UKs still exist, I would tend to remove it.

Reply:
So neither show explicit nor implicit UKs in `show_dm()`?

Reply:
In a new PR?